src/toil/utils/toilStatus.py

```diff
@@ -329,6 +350,8 @@ def main() -> None:
                         help="Print children of each job. default=%(default)s",
                         default=False)
 
+    parser.add_argument("--replayBus", action="store_true",
+                        help="Determine which jobs are currently running and the associated batch system ID")
```

Comment:
Is this the right name for this option? The fact that there's a bus that gets replayed seems like an implementation detail that the user shouldn't have to care about; if the user never hears about the existence of a "message bus", so much the better.

To match the other options I guess this could be `--printBatch` or `--printSchedulerInfo` or `--printJobs` or something. If it wasn't already `toil status` it could be `--printStatus`.

Is there a good reason to make this an option at all, and not just always replay the message bus and print information from it if we can find it?